avocado-setup.py

```diff
@@ -331,7 +331,7 @@ def install_repo(path):
     :param repo: repository path
     """
     logger.info("Installing repo: %s", path)
-    cmd = "cd %s;make requirements;python setup.py install" % path
+    cmd = "cd %s;make requirements;requirements-selftests;python setup.py install" % path
```

Comment:
isn't it `make requirements-selftests` ?

Reply:
yep, that's right, missed it..